### PR TITLE
Veli Ataseven JavaScript Homework Week 1

### PR DESCRIPTION
**Pull request by veliataseven**
Veli Ataseven JavaScript Homework Week 1

### CODE REVIEW

File: Week2/week1/5_variable-z.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+console.log(z);
+
+let a = Math.round(z);
+
+console.log(a);
+
+let highest;
+
+if(a > z)
```
Comment (marcianoviereck) [inline Week2/week1/5_variable-z.js:11]: Nice solution! You can also use [math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max). But this is a fine solution!

File: Week2/week1/8_SAME-TYPE.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+let variableName = [
+    'str1',
+    'num1',
+    'bool',
+    'arr',
+    'str2',
+    'num2'
+];
+
+for(let i = 0; i < (variables.length - 1); i++)
```
Comment (marcianoviereck) [inline Week2/week1/8_SAME-TYPE.js:49]: wow, advanced solution! Really nice!

File: Week2/week1/9_modulo.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+for(let x = 1; x < 8; x++)
```
Comment (marcianoviereck) [inline Week2/week1/9_modulo.js:1]: cool that you know how to use loops already! Will help a lot in future courses

**Review by marcianoviereck — APPROVED** [review]
Really nice solutions! Cool functions,
approved!

**Comment by wouterkleijn** [comment]
Thanks for submitting the homework and trying to do some cool stuff!  @veliataseven

**Comment by veliataseven** [comment]
> Thanks for submitting the homework and trying to do some cool stuff! @veliataseven

You are welcome.. I am trying to do somethings that as much as possible I can

**Pull request closed without merging**